pkg/sinks/http/http.go

```diff
@@ -82,19 +85,20 @@ func NewSink(log logger.Underlying, registry go_metrics.Registry, sink string) (
 }
 
 func (s *HttpSink) Init(ctx context.Context, format formats.Format, compression kt.Compression, fmtr formats.Formatter) error {
-	s.TargetUrl = *TargetUrl
+	for _, url := range strings.Split(*TargetUrl, ",") {
```

Comment:
Should we do a `url.Parse` here to ensure it's a valid target?

Reply:
Good call.